sw/device/lib/crypto/drivers/aes.c

```diff
@@ -199,27 +199,27 @@ aes_error_t aes_decrypt_begin(const aes_key_t key, const aes_block_t *iv) {
 }
 
 aes_error_t aes_update(aes_block_t *dest, const aes_block_t *src) {
-  if (src != NULL) {
-    aes_error_t err = spin_until(AES_STATUS_INPUT_READY_BIT);
+  if (dest != NULL) {
```

Comment:
nit: Does this work if this is the first block being processed, or would it make sense to have a way to check if there is a pending output without necessarily having to use a blocking spin?

Reply:
SG. Thanks